tests/unit/backend/corpora/dataset_processing/test_common.py

```diff
@@ -0,0 +1,208 @@
+import enum
```

Comment:
Moved test here from test_process.py. The only changes are in the imports and some of the mock.patches